tests/cases/mount/mount.jsx

```diff
@@ -0,0 +1,111 @@
+import {findOne} from "../../../dom/traverse";
+import QUnit from "qunit";
+import {mount} from "../../../mount";
+import {h, Component} from "preact";
+
+
+QUnit.module("mount()", {
+    beforeEach: () =>
+    {
+        findOne("#qunit-fixture").innerHTML = `
+            <div id="fixture-1" class="fixture"></div>
+            <div id="fixture-2" class="fixture"></div>
+            <div id="fixture-3" class="fixture"></div>
+        `;
+    },
+});
+
+QUnit.test(
+    "mount() test that the correct parameters are passed, fixes #158",
```

Comment:
The comment `fixes #158` belongs IMO into the commit, not the test case description itself.

```suggestion
    "mount() test that the correct parameters are passed",
```

Reply:
Yeah, I was thinking about that as well. I just removed it.